### ISSUE TITLE
[BUG] Slack Google Maps url has %27 in them, breaking the map

### ISSUE THREAD

**Issue by joberembt** [kvangent/PokeAlarm#155]
## Problem Description

When using slack with the google maps option, the map previews are always out in the middle of the ocean. If I look at the link, it has the url with ' replaced as %27
![chat window](http://i.imgur.com/28jRRby.png)

![url maps](http://i.imgur.com/cSjgnVQ.png)

## Your Environment

Operating System:  Win8.1
Python Version: (use `python -V`)  2.7.10
PokemonGo-Map startup command: `<cmd>`  python runserver.py
PokeAlarm startup command: `<cmd>` python runwebhook,py -l "my location"

Did the issue persist after restarting the script?: yes/no Yes

## Console Output

```
	Nothing useful in console output
```

## Your Alarms Config

```json
    {
			"active": "True",
			"type":"slack",
			"api_key":"GKEY",
			"channel":"CHANNEL",
			"map":{
				"enabled":"True",
				"width":"250",
				"height":"150",
				"far_zoom":"15",
				"maptype":"roadmap"
			}
		},
```

## Troubleshooting Checklist
[X ] Most recent version of PokemonGo-Map
[X ] Most recent version of PokeAlarm
[X] Checked the Wiki page for service in question

**Comment by mrbiggmd** [comment]
I can confirm this issue as well on Ubuntu 14.04.  It was working prior to my pull last night.

**Comment by kvangent** [comment]
It would be great if you could run it in debug and let us see the console output.

**Comment by joberembt** [comment]
Nothing of any real use in the debug log:


python runwebhook.py -l "41.9xxx -91.7xxx" -d
2016-08-25 08:15:48,790 [             utils] [   INFO] Location found: 41.9xxxxx,-91.7xxxxx
2016-08-25 08:15:48,790 [        runwebhook] [   INFO] Debug mode activated!
2016-08-25 08:15:48,806 [     alarm_manager] [   INFO] You will be notified of the following pokemon:
Bulbasaur, Ivysaur, Charmander, Charmeleon, Charizard, Wartortle, Blastoise, Weedle, Pidgey, Ekans, Arbok, Pikachu,
 Raichu, Sandshrew, Sandslash, Nidorina, Nidoqueen, Nidorino, Nidoking, Clefable, Vulpix, Ninetales, Wigglytuff, Gl
oom, Vileplume, Diglett, Dugtrio, Persian, Primeape, Growlithe, Arcanine, Poliwhirl, Poliwrath, Abra, Kadabra, Alak
azam, Machop, Machoke, Machamp, Weepinbell, Victreebel, Tentacool, Tentacruel, Geodude, Graveler, Golem, Ponyta, Ra
pidash, Slowbro, Magnemite, Magneton, Farfetch'd, Doduo, Dodrio, Dewgong, Grimer, Muk, Shellder, Cloyster, Haunter,
 Gengar, Onix, Voltorb, Electrode, Exeggcute, Exeggutor, Cubone, Marowak, Hitmonlee, Hitmonchan, Lickitung, Koffing
, Weezing, Rhyhorn, Rhydon, Chansey, Tangela, Kangaskhan, Seadra, Staryu, Starmie, Mr. Mime, Scyther, Jynx, Electab
uzz, Magmar, Pinsir, Tauros, Magikarp, Gyarados, Lapras, Ditto, Vaporeon, Jolteon, Flareon, Porygon, Omanyte, Omast
ar, Kabuto, Kabutops, Aerodactyl, Snorlax, Articuno, Zapdos, Moltres, Dratini, Dragonair, Dragonite, Mewtwo, Mew
2016-08-25 08:15:48,809 [     alarm_manager] [   INFO] Alarm not activated: pushbullet because value not set to "True"
2016-08-25 08:15:48,819 [    connectionpool] [   INFO] Starting new HTTPS connection (1): slack.com
2016-08-25 08:15:49,674 [    connectionpool] [  DEBUG] "GET /api/channels.list?token=SLACK_TOKEN HTTP/1.1" 200 489
2016-08-25 08:15:49,693 [    connectionpool] [   INFO] Starting new HTTPS connection (1): slack.com
2016-08-25 08:15:49,924 [    connectionpool] [  DEBUG] "GET /api/groups.list?token=SLACK_TOKEN HTTP/1.1" 200 43
2016-08-25 08:15:49,940 [       slack_alarm] [  DEBUG] set([u'CHANNEL1', u'CHANNEL2', u'CHANNEL3'])
2016-08-25 08:15:49,940 [       slack_alarm] [   INFO] Slack Alarm intialized.
2016-08-25 08:15:49,946 [    connectionpool] [   INFO] Starting new HTTPS connection (1): slack.com
2016-08-25 08:15:50,236 [    connectionpool] [  DEBUG] "POST /api/chat.postMessage?token=SLACK_TOKEN HTTP/1.1" 200 216
2016-08-25 08:15:50,243 [             utils] [  DEBUG] REV_LOC set to False
2016-08-25 08:15:50,243 [             utils] [  DEBUG] DM_WALK set to False
2016-08-25 08:15:50,243 [             utils] [  DEBUG] DM_BIKE set to False
2016-08-25 08:15:50,243 [             utils] [  DEBUG] DM_DRIVE set to False
2016-08-25 08:15:50,246 [     alarm_manager] [   INFO] Alarm not activated: twilio because value not set to "True"
2016-08-25 08:15:50,246 [     alarm_manager] [   INFO] Alarm not activated: telegram because value not set to "True"
2016-08-25 08:15:50,249 [     alarm_manager] [   INFO] PokeAlarm has started! Your alarms should trigger now.
2016-08-25 08:15:50,250 [        runwebhook] [   INFO] Webhook server running on http://127.0.0.1:4000
2016-08-25 08:16:10,749 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,753 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.005000
2016-08-25 08:16:10,757 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:10,760 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,763 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.003000
2016-08-25 08:16:10,766 [     alarm_manager] [  DEBUG] Request processing for #19
2016-08-25 08:16:10,766 [     alarm_manager] [   INFO] Rattata ignored: notify not enabled.
2016-08-25 08:16:10,767 [     alarm_manager] [  DEBUG] Finished processing for #19
2016-08-25 08:16:10,770 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,773 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.002000
2016-08-25 08:16:10,776 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:10,779 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,779 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.001000
2016-08-25 08:16:10,788 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,790 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.003000
2016-08-25 08:16:10,796 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:10,799 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,802 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.002000
2016-08-25 08:16:10,805 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:10,808 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,812 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.004000
2016-08-25 08:16:10,813 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:10,818 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,832 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.014000
2016-08-25 08:16:10,836 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:10,838 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,844 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.002000
2016-08-25 08:16:10,846 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:10,848 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,855 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.005000
2016-08-25 08:16:10,858 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:10,861 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:10,862 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:10] "POST / HTTP/1.1" 200 115 0.001000
2016-08-25 08:16:10,865 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:35,713 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:35,716 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:35] "POST / HTTP/1.1" 200 115 0.002000
2016-08-25 08:16:35,717 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:35,723 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:35,726 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:35] "POST / HTTP/1.1" 200 115 0.003000
2016-08-25 08:16:35,733 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:35,736 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:35] "POST / HTTP/1.1" 200 115 0.002000
2016-08-25 08:16:35,736 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:35,743 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:35,743 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:35] "POST / HTTP/1.1" 200 115 0.002000
2016-08-25 08:16:35,746 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:35,750 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:35,753 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:35] "POST / HTTP/1.1" 200 115 0.002000
2016-08-25 08:16:35,753 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:35,759 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:35,759 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:35] "POST / HTTP/1.1" 200 115 0.001000
2016-08-25 08:16:35,762 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
2016-08-25 08:16:35,766 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:35,767 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:35] "POST / HTTP/1.1" 200 115 0.001000
2016-08-25 08:16:35,769 [     alarm_manager] [  DEBUG] Request processing for #16
2016-08-25 08:16:35,772 [     alarm_manager] [   INFO] Pidgey notication was triggered!
2016-08-25 08:16:35,773 [     alarm_manager] [   INFO] metric
2016-08-25 08:16:35,776 [    connectionpool] [   INFO] Starting new HTTPS connection (1): slack.com
2016-08-25 08:16:35,779 [        runwebhook] [  DEBUG] POST request received from 127.0.0.1.
2016-08-25 08:16:35,779 [            pywsgi] [   INFO] 127.0.0.1 - - [2016-08-25 08:16:35] "POST / HTTP/1.1" 200 115 0.001000
2016-08-25 08:16:36,036 [    connectionpool] [  DEBUG] "POST /api/chat.postMessage?token=SLACK_TOKEN HTTP/1.1" 200 316
2016-08-25 08:16:36,051 [     alarm_manager] [  DEBUG] Finished processing for #16
2016-08-25 08:16:36,052 [     alarm_manager] [  DEBUG] Pokestop notifications not yet implemented.
KeyboardInterrupt
Traceback (most recent call last):
  File "runwebhook.py", line 75, in <module>
    server.serve_forever()
  File "c:\python27\lib\site-packages\gevent\baseserver.py", line 349, in serve_forever
    self._stop_event.wait()
  File "c:\python27\lib\site-packages\gevent\event.py", line 207, in wait
    return self._wait(timeout)
  File "c:\python27\lib\site-packages\gevent\event.py", line 117, in _wait
    gotit = self._wait_core(timeout)
  File "c:\python27\lib\site-packages\gevent\event.py", line 95, in _wait_core
    result = self.hub.switch()
  File "c:\python27\lib\site-packages\gevent\hub.py", line 609, in switch
    return greenlet.switch(self)
KeyboardInterrupt
Exception gevent.hub.LoopExit: LoopExit('This operation would block forever', <Hub at 0x3e9b8a0 select default pend
ing=0 ref=0 resolver=<gevent.resolver_thread.Resolver at 0x3f97ef0 pool=<ThreadPool at 0x3f97ed0 0/1/10>> threadpoo
l=<ThreadPool at 0x3f97ed0 0/1/10>>) in <module 'threading' from 'c:\python27\lib\threading.pyc'> ignored

**Comment by kvangent** [comment]
Hopefully this issue is corrected as of commit fd3125ca9ac57626de550f8e9541ba86674a14ef.
